lib/srv/reexec.go

```diff
@@ -153,12 +166,12 @@ func RunCommand() (io.Writer, int, error) {
 	var pty *os.File
 	uaccEnabled := false
 
-	// If a terminal was requested, file descriptor 4 and 5 always point to the
```

Comment:
What's going on here?

The comment referred to fds 4 and 5, but the code referred to fds 5 and 6.

The comment was updated to refer to 5 and 6, but then the code was changed to 6 and 7.

??

Reply:
Ah I fixed the comment first, then ended up incrementing it without adjusting the comment. Fixed.